packages/v2/contracts/ANTv2.sol

```diff
@@ -0,0 +1,171 @@
+pragma solidity 0.5.17;
+
+import './interfaces/IERC20.sol';
+import './libraries/SafeMath.sol';
+
+
+// Lightweight token modelled after UNI-LP: https://github.com/Uniswap/uniswap-v2-core/blob/v1.0.1/contracts/UniswapV2ERC20.sol
+// Adds:
+//   - An exposed `mint()` with minting role
+//   - An exposed `burn()`
+//   - ERC-3009 (`transferWithAuthorization()`)
+contract ANTv2 is IERC20 {
+    using SafeMath for uint256;
+
+    string public constant name = "Aragon Network Token";
+    string public constant symbol = "ANT";
+    uint8 public constant decimals = 18;
+
+    address public minter;
+    uint256 public totalSupply;
+    mapping (address => uint256) public balanceOf;
+    mapping (address => mapping(address => uint256)) public allowance;
+
+    bytes32 public DOMAIN_SEPARATOR;
+
+    // bytes32 public constant PERMIT_TYPEHASH = keccak256("Permit(address owner,address spender,uint256 value,uint256 nonce,uint256 deadline)");
+    bytes32 public constant PERMIT_TYPEHASH = 0x6e71edae12b1b97f4d1f60370fef10105fa2faae0126114a169c64845d6126c9;
+    mapping (address => uint256) public nonces;
+
+    // bytes32 public constant TRANSFER_WITH_AUTHORIZATION_TYPEHASH =
+    //     keccak256("TransferWithAuthorization(address from,address to,uint256 value,uint256 validAfter,uint256 validBefore,bytes32 nonce)");
+    bytes32 public constant TRANSFER_WITH_AUTHORIZATION_TYPEHASH = 0x7c7c6cdb67a18743f49ec6fa9b35f50d52ed05cbed4cc592e13b44501c1a2267;
```

Comment:
I know these constants are related to the storage variables defined right after, but it was harder to read this way, I would place all the constants together with proper comments to provide full context if needed